packages/clerk-js/src/core/clerk.ts

```diff
@@ -1654,6 +1672,16 @@ export class Clerk implements ClerkInterface {
     this.environment = environment;
   }
 
+  public prefetchDependencies = async (): Promise<void> => {
+    if (this.isCoinbaseWalletEnabled) {
+      window.__clerk_coinbase_sdk = this.__promise.promise as Promise<typeof CoinbaseWalletSDK>;
```

Comment:
maybe this need to move inside the constructor.

Also 
```suggestion
      window.__clerk_coinbase_sdk = this.#coinbaseSDKResolvers.promise as Promise<typeof CoinbaseWalletSDK>;
```